### PR TITLE
TASK: Add chapter about contributing to documentation

### PR DESCRIPTION
**Pull request by hhoechtl**

### CODE REVIEW

**Comment by mficzel** [comment]
I like that and will test your docker solution the next days but i am not sure where that piece of documentation should go to. There is already a chapter on the website regarding contribution to documentation. https://www.neos.io/contribute/documenting-neos.html which could also be the correct place. 

Maybe we should start a discussion first wether contribution documentation should go to the docs, to the website or both. If both would be chosen we have to decide how we keep them in sync.

BTW: The contributors appendix is managed from version control somehow as it states in the beginning.

File: TYPO3.Neos/Documentation/Contribute/Documentation/Documentation.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Docker
+======
+
+If you don't want to install Sphinx on your computer or have trouble installing
+it, you can use a prebuilt Docker image that contains a working version of Sphinx.
+The image is built on top of a pretty small alpine linux and has only around 80MB.
+
+You can simply prefix your `make` command with the following docker command:
+
+``docker run -v $(pwd):/documents hhoechtl/doctools-sphinx make html``
```
Comment (mficzel) [inline TYPO3.Neos/Documentation/Contribute/Documentation/Documentation.rst]: The docker image works fine. Thanks a lot for that. It should be mentioned that the command it should be executed inside the Documents directory. 

It should also be explained what the $(pwd):/documents is for. I think that is the path inside the containers where the host files are mounted. Is that right?

Comment (hhoechtl) [inline TYPO3.Neos/Documentation/Contribute/Documentation/Documentation.rst]: As it's a replacement for the `make html` command and that is executed in that directory which is already mentioned in the documentation, I didn't repeat it.

And yes the `$(pwd):/documents` mounts the current directory containing the .rst files into /documents inside the container which is used as entrypoint to generate the files.

**Comment by kitsunet** [comment]
I think it's fine as is and will merge this now. We can change it at any time...

**Pull request merged**